### PR TITLE
Chore: improve wording consistency

### LINKED ISSUE #54: CONSTITUTION.md -- "The Cosmos Hub"
The document frequently mentions "The Cosmos Hub" without clearly specifying whether it pertains to CosmosHub4, AtomOne-Hub, or any hub that follows this constitution presently and in the future.

It is also a possibility that it specifically refers to Cosmos, while some individuals believe that the true hub is the IBC Hub. Considering that forks will inherit the constitution, perhaps we should consider a more inclusive concept?

Should we make any modifications to the document? I suggest considering a different wording or at least adding a definition.

PS: Can you confirm if CONSTITUTION.md should eventually be independent or if we should always consider it in conjunction with README.md? Because README.md is undoubtedly superior for definitions.

### PR DESCRIPTION
**Pull request by moul**
Fixes #54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ use $ATOMs; by offering what is misleadingly referred to as "liquid staking".
 
 ## Role as $ATOM "Liquid Staking" Provider
```
Comment (giunatale) [inline README.md:137]: In this section we need to be careful with the renaming of $phATOM to $PHOTON.

This section talks about the "$ATOM-version" of $PHOTON. We should disambiguate if we want to use the same name while we work on the final design/naming scheme.

**Review by giunatale — COMMENTED** [review]
from https://github.com/atomone-hub/genesis/issues/54#issuecomment-1839272695

and looking at the changes, my understanding is you are using $PHOTON for the $ATOM1 version and $phATOM1 for the $ATOM version, correct?

(as done e.g. at line 876)

File: README.md (hunk 1 above)
Comment (moul) [reply README.md:137]: Thx, if you spot additional ones, feel free to share.

I’ll fix and make a second review soon.

File: README.md [hunk 2]
```diff
@@ -873,7 +876,7 @@ available for hosting on AtomOne ICS and its successors.
  - [ ] Scan through twitter posts for more ideaas.
  - [ ] Argument for why hub and spokes are needed (from atom one)
  - [ ] Quantum resistance
- - [ ] Constitution updates: $ATOM -> $ATOM1; Add $phATOM and $phATOM1; conversion
+ - [ ] Constitution updates: $ATOM -> $ATOM1; Add $PHOTON and $phATOM1; conversion
```
Comment (jaekwon) [inline README.md:879]: Add $phATOM and $PHOTON;
$ATOM -> $phATOM.
$ATONE -> $PHOTON.

**Pull request merged**